src/R/Editor/Impl/Commands/RMainController.cs

```diff
@@ -33,7 +33,7 @@ public RMainController(ITextView textView, ITextBuffer textBuffer)
         /// <summary>
         /// Retrieves R command controller from text view
         /// </summary>
-        public static RMainController FromTextView(ITextView textView) {
+        public static new RMainController FromTextView(ITextView textView) {
```

Comment:
Ok

---
In reply to: [64487288](https://github.com/Microsoft/RTVS/pull/1812#discussion_r64487288) [](ancestors = 64487288)